src/zksync/bridge-utils/sendRawTransactionWithDetailedOutput.ts

```diff
@@ -0,0 +1,51 @@
+import type { Address } from 'abitype'
+import type { Client } from '../../clients/createClient.js'
+import type { Transport } from '../../clients/transports/createTransport.js'
+import type { Account } from '../../types/account.js'
+import type { Chain } from '../../types/chain.js'
+import type { Hash, Hex } from '../../types/misc.js'
+import type { PublicZkSyncRpcSchema } from '../types/eip1193.js'
+
+export type SendRawTransactionWithDetailedOutputParameters = {
+  signedTx: Hex
```

Comment:
```suggestion
  serializedTransaction: Hex
```

Keep this consistent with Viem's `sendRawTransaction`